server/bleep/src/semantic/schema.rs

```diff
@@ -40,3 +48,18 @@ impl PartialEq for Payload {
         // created payload
     }
 }
+
+pub(super) fn create_collection(name: &str) -> CreateCollection {
```

Comment:
minor nit: can we have this share a signature similar to `create_indexes`: accept a reference to a qdrant client?